Client/sdk/game/CModelInfo.h

```diff
@@ -28,6 +28,27 @@ class CBoundingBox
     float   fRadius;
 };
 
+enum class eModelIdeFlag
+{
+    IS_ROAD,
+    DRAW_LAST,
+    ADDICTIVE,
+    IGNORE_LIGHTING,            // Used with animated objects
+    NO_ZBUFFER_WRITE,
+    DONT_RECEIVE_SHADOWS,
+    IS_GLASS_TYPE_1,
+    IS_GLASS_TYPE_2,
+    IS_GARAGE_DOOR,
+    IS_DAMAGABLE,
+    IS_TREE,
+    IS_PALM,
+    DOES_NOT_COLLIDE_WITH_FLYER,
+    IS_TAG,
+    DISABLE_BACKFACE_CULLING,
+    IS_BREAKABLE_STATUE,
+    IS_CRANE,
+};
+
```

Comment:
Perhaps make this an `enum` and use it as a bit mask?
Or hack it like this, so nobody can access it like `IS_TAG`:
```cpp
namespace eModelIdeFlag
{
enum
{
    IS_ROAD,
    DRAW_LAST,
    ADDICTIVE,
    IGNORE_LIGHTING,            // Used with animated objects
    NO_ZBUFFER_WRITE,
    DONT_RECEIVE_SHADOWS,
    IS_GLASS_TYPE_1,
    IS_GLASS_TYPE_2,
    IS_GARAGE_DOOR,
    IS_DAMAGABLE,
    IS_TREE,
    IS_PALM,
    DOES_NOT_COLLIDE_WITH_FLYER,
    IS_TAG,
    DISABLE_BACKFACE_CULLING,
    IS_BREAKABLE_STATUE,
    IS_CRANE,
};
};
```

Reply:
Yeah,  i have think about bit mask but i can't use it for some flags ( IS_DAMAGABLE, IS_TREE, IS_GLASS_TYPE_2, ... )